modules/core/src/lib/animation-loop.js

```diff
@@ -164,6 +167,7 @@ export default class AnimationLoop {
   // Redraw now
   redraw() {
     this._beginTimers();
+    this.timeline.update(Date.now() - this.animationProps.startTime);
```

Comment:
- `this.timeline.update(Date.now())` should suffice?
- Should this be inside `_updateCallbackData`?

Reply:
Yes, for the first item. I'll leave it outside _updateCallbackData for now because I'm imagining it as a separate system from the render props. But it could go in there if that ends up being the primary usage.

Reply:
Actually, I decided to do the opposite! I've moved the update into _updateCallbackData, since the other time updates are there. But I decided to keep using the time since startup to update, because it provides a bit of a safeguard against float precision issues.